src/Components/Endpoints/src/Binding/FormDataMapperOptions.cs

```diff
@@ -16,8 +16,16 @@ public FormDataMapperOptions()
 
         _factories.Add((type, options) => ParsableConverterFactory.Instance.CanConvert(type, options) ? ParsableConverterFactory.Instance.CreateConverter(type, options) : null);
         _factories.Add((type, options) => NullableConverterFactory.Instance.CanConvert(type, options) ? NullableConverterFactory.Instance.CreateConverter(type, options) : null);
+        _factories.Add((type, options) => CollectionConverterFactory.Instance.CanConvert(type, options) ? CollectionConverterFactory.Instance.CreateConverter(type, options) : null);
```

Comment:
I realize this pattern was added in a previous PR, but why are there two separate calls here instead of a single `TryCreateConverter()` method that checks if a converter can be created, then creates one if possible? Could that eliminate any duplication between implementations of those two methods?

Reply:
It's following the same patterns as S.T.J in some cases the check for whether the converter can create the type is much simpler than creating the converter itself.